infra/terraform/modules/data_access/teams/teams.py

```diff
@@ -213,4 +214,19 @@ def detach_policy_from_user(policy_arn, user_name):
 
 
 def bucket_name(slug):
```

Comment:
I'll address the duplication separately: https://trello.com/c/ZbjN4qxy/234-s-factor-out-lambdas-duplicated-code